### PR TITLE
Add payload on Get and List

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

### CODE REVIEW

**Comment by greut** [comment]
the issue is with e.g. `VirtualMachine.Get`

**Comment by greut** [comment]
where the `ErrorText` could be better if it contained the actual query made: https://github.com/exoscale/egoscale/blob/master/virtual_machines.go#L21-L43

**Comment by pierre-emmanuelJ** [comment]
here an exemple

**Comment by pierre-emmanuelJ** [comment]
get on unknown vm name
```
2018/06/05 10:27:32 Unable to get Virtual Machine "fdqsfds". API error ParamError 431 (CSErrorCode(0) 0): missing VirtualMachine. id: , name: fdqsfds apikey=EXOd780edbb1f385c827853e5f4&command=listVirtualMachines&response=json
```

File: virtual_machines.go [hunk 1]
```diff
@@ -29,14 +29,19 @@ func (vm *VirtualMachine) Get(ctx context.Context, client *Client) error {
...
 	count := len(vms)
 	if count == 0 {
 		return &ErrorResponse{
 			ErrorCode: ParamError,
-			ErrorText: fmt.Sprintf("missing VirtualMachine. id: %s, name: %s", vm.ID, vm.Name),
+			ErrorText: fmt.Sprintf("missing VirtualMachine. id: %s, name: %s %s", vm.ID, vm.Name, payload),
 		}
 	} else if count > 1 {
-		return fmt.Errorf("more than one VirtualMachine was found. Query: id: %s, name: %s", vm.ID, vm.Name)
+		return fmt.Errorf("more than one VirtualMachine was found. Query: id: %s, name: %s %s", vm.ID, vm.Name, payload)
```
Comment (greut) [inline virtual_machines.go]: As `id` and `name` are not always relevant, we may only show the payload.
`"missing VirtualMachine. query: %s", payload`

Comment (greut) [reply virtual_machines.go]: once we have that, is there a way to refactor the `Get` call?

**Comment by greut** [comment]
@pierre-emmanuelJ `Get` duplication is gone.

**Comment by pierre-emmanuelJ** [comment]
I like this way for gettable ! for me you can merge

**Pull request merged**